imageroot/systemd/user/passbolt-app.service

```diff
@@ -11,7 +11,7 @@ Restart=always
 TimeoutStopSec=70
 ExecStartPre=/bin/rm -f %t/passbolt-app.pid %t/passbolt-app.ctr-id
 ExecStartPre=-runagent discover-smarthost
-ExecStart=/usr/bin/podman run --conmon-pidfile %t/passbolt-app.pid --cidfile %t/passbolt-app.ctr-id --cgroups=no-conmon --pod-id-file %t/passbolt.pod-id --replace -d --name passbolt-app --env-file=%S/state/config.env --env-file=%S/state/smarthost.env -v passbolt-gpg:/etc/passbolt/gpg -v passbolt-jwt:/etc/passbolt/jwt ${PASSBOLT_IMAGE}
+ExecStart=/usr/bin/podman run --conmon-pidfile %t/passbolt-app.pid --cidfile %t/passbolt-app.ctr-id --cgroups=no-conmon --pod-id-file %t/passbolt.pod-id --replace -d --name passbolt-app --env-file=%S/state/config.env --env-file=%S/state/smarthost.env -v passbolt-gpg:/etc/passbolt/gpg:Z -v passbolt-jwt:/etc/passbolt/jwt:Z ${PASSBOLT_IMAGE}
```

Comment:
could you explode the lines with `\`

also the selinux context should be adapted to `:z` in fact in a pod context every container is allowed to  access to a volume of a container, it avoids to change all the permission of a volume. @DavidePrincipi gave me this hint recently

Reply:
could be changed to --volume